certbot-apache/certbot_apache/constants.py

```diff
@@ -62,9 +62,9 @@
     vhost_files="*.conf",
     logs_root="/var/log/apache2",
     version_cmd=['/usr/sbin/apache2', '-v'],
-    define_cmd=['apache2ctl', 'virtualhosts'],
-    restart_cmd=['apache2ctl', 'graceful'],
-    conftest_cmd=['apache2ctl', 'configtest'],
+    define_cmd=['/usr/sbin/apache2', '-S'],
```

Comment:
I know that it is labeled the default but as I said ans you should know (since you have installed Gentoo yourself), there is nothing installed by default, you chose to install ObenRC.

I also know, that it is no rocket science (I too have each vhost in a separate file) but by insisting on using the autoconfigure feature (instead of doing the configuration yourself), you deny everyone with systemd the possibility to use certbot at all.